modules/ROOT/pages/design-import-files.adoc

```diff
@@ -52,7 +50,7 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 +
 *Version*:: The version of a specification or fragment. Select the *Include snapshots* check box in the bottom-left corner of the dialog to include snapshots in this column.
 +
-*Type*:: The type of asset. The value can be either *API Specification* or *RAML Fragment*.
+*Type*:: The type of asset. The value can be either *REST API* or *API Fragment*.
```

Comment:
```suggestion
*Type*:: The type of asset
The value can be either *REST API* or *API Fragment*.
```

Reply:
I understand that implementing this reformatting might not be possible at this point.
But please note that it is a global standard documented in the style ref, under the list entries.